integrations/azure_ai_search/tests/test_document_store.py

```diff
@@ -157,8 +160,8 @@ def test_from_dict_with_params(monkeypatch):
             "index_name": "my_index",
             "embedding_dimension": 15,
             "metadata_fields": {
-                "Title": "str",
-                "Pages": "int",
+                "Title": SimpleField(name="Title", type="Edm.String", filterable=True).as_dict(),
```

Comment:
As in the previous test, lets keep the `Title` as a `SearchField` for more coverage.

```suggestion
                "Title": SearchField(name="Title", type="Edm.String", filterable=True).as_dict(),
```